addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -1178,43 +1178,42 @@ def test_in_invoice_create_refund(self):
         self.assertInvoiceValues(reverse_move, [
```

Comment:
```suggestion
        self.assertInvoiceValues(new_invoice, [
```
The old variable name doesn't make sense anymore and creates confusion (several occurrences).